### PR TITLE
rgw: bucket stats report mtime in UTC

### PR DESCRIPTION
**Pull request by cbodley**
Fixes: https://tracker.ceph.com/issues/39295

### CODE REVIEW

File: src/rgw/rgw_bucket.cc [hunk 1]
```diff
@@ -1452,7 +1452,7 @@ static int bucket_stats(RGWRados *store, const std::string& tenant_name, std::st
   ::encode_json("owner", bucket_info.owner, formatter);
   formatter->dump_string("ver", bucket_ver);
   formatter->dump_string("master_ver", master_ver);
-  formatter->dump_stream("mtime") << ut;
+  ut.gmtime(formatter->dump_stream("mtime"));
```
Comment (mattbenjamin) [inline src/rgw/rgw_bucket.cc]: wierd locution

Comment (cbodley) [reply src/rgw/rgw_bucket.cc]: agreed, but this is used elsewhere in rgw. both gmtime() and localtime() take a `std::ostream&`, the only difference is that `operator<<(utime_t)` calls localtime() directly

**Comment by LenzGr** [comment]
Thanks for the quick resolution! LGTM.

**Comment by ivancich** [comment]
Pulpito results:
http://pulpito.ceph.com/ivancich-2019-04-23_19:53:03-rgw-wip-eric-testing-1-distro-basic-smithi/
http://pulpito.ceph.com/ivancich-2019-04-23_19:52:23-rgw-wip-eric-testing-1-distro-basic-smithi/

File: src/rgw/rgw_bucket.cc (hunk 1 above)
Comment (mattbenjamin) [reply src/rgw/rgw_bucket.cc]: ack, ok

**Pull request merged**